src/ColorVectorSpace.jl

```diff
@@ -293,7 +292,7 @@ dotc(x::T, y::T) where {T<:AbstractRGB} = 0.200f0 * acc(red(x))*acc(red(y)) + 0.
 dotc(x::AbstractRGB, y::AbstractRGB) = dotc(promote(x, y)...)
 
 # Scalar Gray
-const unaryOps = (:~, :conj, :abs,
+const UnaryOps = (:~, :conj, :abs,
```

Comment:
Since this isn't a type or module I'd propose lowercase.  (Yes, the dromedarycase was weird.)

Reply:
Done !